mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/Connection.cs

```diff
@@ -215,6 +216,7 @@ class ValueImpl {
 		public bool IsEnum; // For ElementType.ValueType
 		public long Id; /* For VALUE_TYPE_ID_TYPE */
 		public int Index; /* For VALUE_TYPE_PARENT_VTYPE */
+		public int FixedSize = 1; 
 	}
```

Comment:
Why does this need to be inited to 1 ?

Reply:
We don't need anymore, we need this before when I used the same method for both cases.